app/swift_demangle.c

```diff
@@ -0,0 +1,147 @@
+
```

Comment:
missing copyright header, did you write all this or was this copied from somewhere?

Reply:
I wrote it. Happy to put it under the same license as the rest of perfparser. Is there anything you need me to do for that? Put a GPL license header at the top?

Reply:
yeah exactly, just copy what the other files are doing and put your name in